docs/ir.rst

```diff
@@ -400,7 +400,7 @@ convention:
     retlist      : paramlist
     param        : type [paramext] [paramspecial]
     paramext     : "uext" | "sext"
-    paramspecial : "sret" | "link" | "fp" | "csr" | "vmctx"
+    paramspecial : "sret" | "link" | "fp" | "csr" | "vmctx" | "sigid" | "stack_limit"
```

Comment:
Is it expected that the new `sigid` special parameter is added to the list, but never used in any definition? I'd expect to see it shown in the definition of a call instruction which can call into a table_addr.

Reply:
I'm unclear on what you mean here. Are you looking for an example of how to use `sigid`?

Reply:
Yes indeed, thanks for checking; other special parameters I've checked in the docs have a usage example, this one does not.

Reply:
Ok, I opened https://github.com/CraneStation/cranelift/issues/497 to track the lack of an example for how to use `sigid`.